src/tiled/commandlineparser.cpp

```diff
@@ -118,13 +116,10 @@ bool CommandLineParser::parse(const QStringList &arguments)
 
 void CommandLineParser::showHelp()
 {
-    // TODO: Make translatable
-    qWarning().nospace() << "Usage:\n"
-                         << "  " << qPrintable(mCurrentProgramName)
-                         << " [options] [files...]\n\n"
-                         << "Options:";
+    qWarning().nospace() << qPrintable(tr("Usage:\n  %1 [options] [files...]").arg(mCurrentProgramName))
+                         << "\n\n" << tr("Options") << ":";
```

Comment:
Hmm, I may have missed this before, but the `tr("Options")` also needs to be wrapped in `qPrintable`.